docs/development_suite/user-feedback/give-feedback.md

```diff
@@ -0,0 +1,31 @@
+---
+id: give-feedback
+title: Give Feedback
+sidebar_label: Give Feedback
+---
+
+Mia-Platform users can actively contribute to the product through a feature that allows them to submit feedback on any area of the Console. This feature is accessible through a dedicated button located on the right side of the Console top bar.
```

Comment:
```suggestion
Mia-Platform users can actively contribute to the product through a feature that allows them to submit feedback on any area of the Console. This feature is accessible through a dedicated button located on the right side of the Console top bar. It is a useful tool to understand the sentiment of your users
```

Reply:
Maaa in che senso? È uno strumento utile **A NOI** per capire il sentiment degli utenti, ma non per gli utenti stessi. Per loro l'utilità è data dal fatto che possono contribuire con delle proposte di miglioramento 😬